lib/fastlane/plugin/firebase_app_distribution/actions/firebase_app_distribution_action.rb

```diff
@@ -29,32 +29,34 @@ def self.run(params)
         auth_token = fetch_auth_token(params[:service_credentials_file], params[:firebase_cli_token])
         fad_api_client = Client::FirebaseAppDistributionApiClient.new(auth_token, params[:debug])
 
-        # If binary is an AAB get FULL view of app which includes the aab_state
-        app_view = binary_type == :AAB ? 'FULL' : 'BASIC'
-        app = fad_api_client.get_app(app_id, app_view)
-        validate_app!(app, binary_type)
-        release_id = fad_api_client.upload(app.project_number, app_id, binary_path, platform.to_s)
+        # If binary is an AAB, get the AAB info for this app, which includes the integration state and certificate data
+        if binary_type == :AAB
+          aab_info = fad_api_client.get_aab_info(app_id)
```

Comment:
would be nice if all methods would be changed to work with resource names rather than IDs, so `app_id` would be replaced by `app_name` and `app_id`+`release_id` would just be `release_name`

Reply:
Done!